webpack.config.js

```diff
@@ -43,6 +43,15 @@ module.exports = (argv) => {
           test: /\.(woff|woff2|eot|ttf|otf)$/i,
           type: 'asset/resource',
         },
+        {
+          test: /\.(png|svg|jpg|jpeg|gif|webp)$/i,
+          use: ['file-loader', 'webp-loader'],

```

Comment:
Is this web-p loader needed? Can't it be done via `type: 'asset/resource'` ?

Reply:
Changed it to `type: 'asset/resource'`.